src/test/unit/wrappers/http/test_sync_http_wrappers_threads.py

```diff
@@ -29,3 +32,25 @@ def lambda_test_function(event, context):
             "my_index"
         ]
         assert request_index == response_index
+
+
+def test_run_in_executor(context, token):
+    @lumigo_tracer.lumigo_tracer(token=token)
+    def lambda_test_function(event, context):
+        async def main():
+            loop = asyncio.get_event_loop()
+            future1 = loop.run_in_executor(None, requests.get, "http://www.google.com")
+            response1 = await future1
```

Comment:
I think it's worth doing 2 API calls (with async.gather) to validate that it really works with concurrency.